internal/clients/recommendations/client.go

```diff
@@ -0,0 +1,168 @@
+//go:generate go run -mod=mod github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen --config client.cfg.yml recommendations.v3.json
+package recommendations
```

Comment:
Please move this comment into a separate file called `package.go` like it is done in https://github.com/osbuild/image-builder/pull/1128

But only if you get your tests red, in case you are green let’s merge this and I will update my PR :-)